### PR TITLE
Add WCAG 2.1 support to color contrast dialog

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Describe the change
This adds non-text element support (new for WCAG 2.1) to the color contrast analyzer, while also refactoring the dialog to the latest spec.

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [x] Includes UI changes?
  - [x] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [x] Attach any screenshots / GIF's that are applicable.

##### High Contrast Disabled:
image

##### High Contrast White:
image

##### High Contrast Black:
image

##### High Contrast 1:
image

##### High Contrast 2:
image

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

**Comment by peterdur** [comment]
Is it expected that the ratio next to the eyedroppers can receive keyboard focus? TAB from the new WCAG 1.4.11 link goes there.
image

**Comment by flyingUnderTheRadar** [comment]
> Is it expected that the ratio next to the eyedroppers can receive keyboard focus? TAB from the new WCAG 1.4.11 link goes there.
> image
> **Edit:** Looks like this happens in the current production version.
Was it shift + tab that got you here @peterdur. (It's the last thing that gets focus when using tab) I believe that its made focusable so that a screen reader would be able to read it.

**Comment by jalkire** [comment]
> Is it expected that the ratio next to the eyedroppers can receive keyboard focus? TAB from the new WCAG 1.4.11 link goes there.
> image
> **Edit:** Looks like this happens in the current production version.
> Was it shift + tab that got you here @peterdur. (It's the last thing that gets focus when using tab) I believe that its made focusable so that a screen reader would be able to read it.

Yes, this was/is intended behavior.

**Comment by peterdur** [comment]
> Yes, this was/is intended behavior.

OK

File: src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml [hunk 1]
```diff
@@ -298,6 +301,106 @@
                            Background="{Binding Path=ContrastVM.FirstColor, Converter={converters:ColorStringConverter}, RelativeSource={RelativeSource Mode=FindAncestor,AncestorType=local:ColorContrast}}"
                            AutomationProperties.Name="{x:Static Properties:Resources.ColorContrast_LargeSampleHelpText}" HorizontalAlignment="Left"/>
                     </StackPanel>
+                    <Border Grid.Row="4" Grid.ColumnSpan="4" BorderBrush="{DynamicResource ResourceKey=GreyBackgroundBrush}"
+                        BorderThickness="0,0,0,1" Background="{x:Null}" />
+                    <StackPanel Orientation="Horizontal" Grid.Row="5" Grid.Column="1" Margin="4,4">
+                        <fabric:FabricIconControl x:Name="graphicalObjectGlyph" VerticalAlignment="Center" Margin="0 0 2 0">
+                            <fabric:FabricIconControl.Style>
+                                <Style TargetType="fabric:FabricIconControl" BasedOn="{StaticResource GlyphToForeground}">
```
Comment (jalkire) [inline src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml]: Might be worth pulling this and below style out into `styles.xaml`--we usually keep them there.

Comment (DaveTryon) [reply src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml]: As discussed offline, I'll shoot for the refactor in a separate PR

**Pull request merged**